installer/msi/BinaryFragment.wxs

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2020-Present MongoDB Inc. -->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <DirectoryRef Id="BIN">
+	    <?if $(var.Platform)=x64 ?>
+	       <?define win64Flag = "yes" ?>
+	    <?else ?>
+	       <?define win64Flag = "no" ?>
+	    <?endif ?>
+       <Component Guid="{02DA3AB6-9773-4B13-9D02-7874FE99050D}" Id="RegKeys" Win64="$(var.win64Flag)">
```

Comment:
this is the same UUID that we're using for the ODBC driver installer. Is that what we're supposed to be doing?

Reply:
It was not, thanks for catching that.